src/main/java/net/wasdev/wlp/ant/AbstractTask.java

```diff
@@ -427,6 +428,88 @@ protected String findStringInFile(String regexp, File fileToSearch) throws Excep
         return matches;
     }
 
+    /**
+     * Searches the given file for the given regular expression.
+     *
+     * @param regexp
+     *            a regular expression (or just a text snippet) to search for
+     * @param fileToSearch
+     *            the file to search
+     * @return List of Strings which match the pattern. No match results in an
+     *         empty list.
+     * @throws Exception
+     */
+    protected List<String> findStringsInFileCommon(String regexp,
+                                                   boolean stopOnFirst, int searchLimit, File fileToSearch)
+                    throws Exception {
```

Comment:
As stated above, I don't see the need for this method to throw an exception.

Reply:
done